requirements.txt

```diff
@@ -1,9 +1,10 @@
-simpy==3.0.11
-networkx==2.5
-numpy==1.18.5
-matplotlib==3.2.2
-pandas==1.0.5
-seaborn==0.10.1
-scipy==1.4.1
-dotmap==1.2.20
+dotmap==1.3.30
 exmas==0.9.99
+matplotlib==3.6.0
```

Comment:
to jest spojne? dzialajace?
Mialem tam jakies problemy kiedys, stad konkretne wersje (==) ale to co teraz tes jest stabilne?

Potem tez przy tf i pytorch sie robia jeszcze wieksze problemy, sczegonie na macu M1.

Reply:
Przed mergem z upstreama udało mi się zreprodukować działanie wszystkich testów, konfiguracji oraz mojego środowiska na Pythonie 3.8 uruchomionym w Condzie na MacBooku Pro z Intelem i7. Ciężko tutaj zapewnić stabilność, szczególnie jeśli mówimy o innych architekturach procesora poza ręcznym uruchomieniem na konkretnej maszynie. Alternatywą jest CI/CD które uruchamia się również na zewnętrznych maszynkach o docelowych konfiguracjach (np. AWS oferuje instancje z procesorami z rodziny Apple Silicon https://aws.amazon.com/about-aws/whats-new/2022/07/general-availability-amazon-ec2-m1-mac-instances-macos/) tylko to bardzo droga sprawa.